sc2reader/objects.py

```diff
@@ -285,6 +285,21 @@ def __init__(self, uid, init_data):
         #: This is deprecated because it doesn't actually work.
         self.recorder = None
 
+        #: The user's mmr at the time of the game
```

Comment:
What is `mmr` and why does the reader need to guess?

Reply:
MMR stands for Matchmaking Rating, indicating the skill level on SC2 ladder. I assume this is a common abbreviation like APM under sc2 context, but now I dont know whether we should explain it in comment or not.